nexus/src/app/disk.rs

```diff
@@ -54,6 +54,30 @@ impl super::Nexus {
                         ),
                     });
                 }
+
+                // Reject disks where the size isn't at least
+                // MIN_DISK_SIZE_BYTES
+                if params.size.to_bytes() < params::MIN_DISK_SIZE_BYTES as u64 {
+                    return Err(Error::InvalidValue {
+                        label: String::from("size"),
+                        message: String::from(
+                            "total size must be at least MIN_DISK_SIZE_BYTES",
```

Comment:
This error would be okay if this was just an internal error, but since this is user-facing, it might be a little harder to decipher. 

I went ahead and made https://github.com/oxidecomputer/omicron/pull/1171 ; this should make it possible to just:
1. Convert MIN_DISK_SIZE_BYTES to `ByteCount`
2. Display that directly

Reply:
Once this is committed I'll make sure to implement. Thanks, Sean.